src/ripple/net/DatabaseBody.h

```diff
@@ -114,9 +120,9 @@ class DatabaseBody::reader
 {
     value_type& body_;  // The body we are writing to
 
-    static const uint32_t FLUSH_SIZE = 50000000;
-    static const uint8_t MAX_HANDLERS = 3;
-    static const uint16_t MAX_ROW_SIZE_PAD = 500;
+    static const std::uint32_t FLUSH_SIZE = 50000000;
+    static const std::uint8_t MAX_HANDLERS = 3;
+    static const std::uint16_t MAX_ROW_SIZE_PAD = 500;
```

Comment:
Is it better because its guaranteed to be set at compile time?

Reply:
Done

Reply:
With integers the difference between `static const` and `static constexpr` is unlikely to be noticeable.  But it's a good habit to get into since it does make a difference when the object has a non-trivial constructor.